src/app.js

```diff
@@ -42,7 +42,10 @@ app.use(compress())
 	.options('*', cors())
 	.use(cors())
 	.use(favicon(path.join(app.get('public'), 'favicon.ico')))
-	.use('/', serveStatic(app.get('public')))
+//	.use('/', serveStatic(app.get('public')))
+	.use('/',(req,res)=>{
+		res.status(308).redirect('/docs/');
```

Comment:
so `public/index.html` is not used anymore?

Reply:
Yes that is to be achieved. But do not work at the moment.